dts/arc/quark_se_c1000_ss.dtsi

```diff
@@ -0,0 +1,50 @@
+#include "skeleton.dtsi"
+#include "mem.h"
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arc";
+			reg = <1>;
+		};
+	};
+
+	flash0: flash@40010000 {
+		reg = <0x40000000 DT_FLASH_SIZE>;
+	};
+
+	sram0: memory@a8000400 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+		reg = <0xa8000400 DT_SRAM_SIZE>;
+	};
+
+
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "simple-bus";
+		ranges;
+
+		uart0: uart@b0002000 {
+			compatible = "intel,qmsi_ss-uart";
```

Comment:
Seems like we are using same driver, and not a SS specific one, so any reason this shouldn't just be:

"intel,qmsi-uart"

Reply:
yes. I will correct it